tests/integrationv2/providers.py

```diff
@@ -179,9 +182,15 @@ def setup_client(self, options: ProviderOptions):
                 pytest.skip('BoringSSL does not support Cipher {}'.format(options.cipher))
             elif options.cipher == Ciphersuites.TLS_AES_256_GCM_384:
                 pytest.skip('BoringSSL does not support Cipher {}'.format(options.cipher))
+        if options.curve is not None:
+            if options.curve == Curves.P256:
+                cmd_line.extend(['-curves', 'P-256'])
+            if options.curve == Curves.P384:
+                cmd_line.extend(['-curves', 'P-384'])
```

Comment:
BoringSSL curently doesn't support X25519 as well, do we want to add a place holder `if` condition with a `pytest.skip` as done above for ciphers not supported.

Reply:
Good call. It isn't failing right now because X25519 isn't in the configured curves list. I've added the `pytest.skip` call.